sdoc.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = ["README.md"]
 
   s.add_runtime_dependency("rdoc", "~> 5.0")
+  s.add_runtime_dependency("actionview", "~> 4.2")
```

Comment:
This doesn't allow 5+, so it may be a problem in the Rails repository. But why you need action view?